macOS/DuckDuckGoVPN/VPNProxyLauncher.swift

```diff
@@ -88,7 +99,7 @@ final class VPNProxyLauncher {
 
     private var isControllingProxy = false
 
-    private func startOrStopProxyIfNeeded(isProxyConnectionStatusChange: Bool = false) async throws {
+    private func startOrStopProxyIfNeeded(isProxyStatusChange: Bool = false) async throws {
```

Comment:
Small change to improve readability.  The previous name is just long and awful to read.